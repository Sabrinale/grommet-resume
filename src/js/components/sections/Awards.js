import React from 'react'

import { formatDate } from '../common'
import { SectionTitle } from '../SectionTitle'
import { SecondaryTitle } from '../SecondaryTitle'
import { Description } from '../Description'
import { Highlights } from '../Highlights'
import { DateSection } from '../DateSection'

const Award = ({ item }) => (
  <Highlights>
    <SectionTitle>{item.title}</SectionTitle>
    <SecondaryTitle>{item.awarder}</SecondaryTitle>
    <DateSection>{formatDate(item.date)}</DateSection>
    <Description>{item.summary}</Description>
  </Highlights>
)

export { Award }
