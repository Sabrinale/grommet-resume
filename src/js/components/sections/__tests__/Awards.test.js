import React from 'react'
import renderer from 'react-test-renderer'

import { Award } from '../Awards'

import defaultResume from './mockResume.json'

test('renders awards section', () => {
  const data = defaultResume.awards[0]
  const component = renderer.create(<Award item={data} />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
