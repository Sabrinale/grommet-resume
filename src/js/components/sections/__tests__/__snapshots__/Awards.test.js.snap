// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders awards section 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 kFdcfG"
>
  <h3
    className="StyledHeading-sc-1rdh4aw-0 jVEgcZ"
  >
    Award
  </h3>
  <h4
    className="StyledHeading-sc-1rdh4aw-0 hlZfma"
  >
    Company
  </h4>
  <span
    className="StyledText-sc-1sadyjn-0 dndifk"
    size="medium"
  >
    Nov 2014
  </span>
  <p
    className="StyledParagraph-tbetod-0 jzEULC"
  >
    There is no spoon.
  </p>
</div>
`;
